invenio_cli/cli/services.py

```diff
@@ -36,7 +36,7 @@ def start(cli_config):
 @services.command()
 @click.option('-f', '--force', default=False, is_flag=True,
               help='Force recreation of db tables, ES indices, queues...')
-@click.option('-n', '--no-demo-data', default=False, is_flag=True,
+@click.option('-N', '--no-demo-data', default=False, is_flag=True,
```

Comment:
Nice catch! could you also change it in the [containers](https://github.com/inveniosoftware/invenio-cli/blob/9227f0607aab925a7d02a7134b54a02314409af1/invenio_cli/cli/containers.py#L44) commands? cuz it happens too. Thanks!

Reply:
:100: